core/pl/util.pm.sdoc

```diff
@@ -90,3 +90,8 @@ sub wf {
   close $fh;
   $_[0];
 }
+
+sub testpath {
+  $_ =~ s/-\*/-0000\*/;
```

Comment:
My mistake -- I had been thinking of [this one](https://github.com/spencertipping/ni/blob/master/core/boot/util.pl.sdoc#L8), but I realized that it doesn't get exported to perl mappers specifically because of that name collision.